pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -502,3 +505,69 @@ func checkForPodSecondaryNetworkAttachement(pod *corev1.Pod) (string, bool) {
 		return netObj, false
 	}
 }
+
+// RestoreSecondaryNetwork restore cniCache and pod interface when agent restart.
+func (pc *PodController) RestoreSecondaryNetwork(ovsBridgeClient ovsconfig.OVSBridgeClient, ifaceStore interfacestore.InterfaceStore) {
+	err := InitializeInterfaceStore(ovsBridgeClient, ifaceStore)
+	if err != nil {
+		klog.ErrorS(err, "Failed to initialize the secondary bridge interface store")
+	}
+
+	knownInterfaces := ifaceStore.GetInterfacesByType(interfacestore.ContainerInterface)
+	for _, containerConfig := range knownInterfaces {
+		Config, exists := ifaceStore.GetContainerInterface(containerConfig.ContainerID)
+		if !exists {
+			klog.InfoS("The container interface has been deleted ", "container id", Config.ContainerID)
+			return
+		}
+		event := types.PodUpdate{
+			IsAdd:        true,
+			PodName:      Config.PodName,
+			PodNamespace: Config.PodNamespace,
+			ContainerID:  Config.ContainerID,
+		}
+		pc.processCNIUpdate(event)
+
+	}
+}
+
+func InitializeInterfaceStore(ovsClient ovsconfig.OVSBridgeClient, ifaceStore interfacestore.InterfaceStore) error {
```

Comment:
Why not use PodController's signature? e.g.,
```
func (pc *PodController) initializeInterfaceStore() error{
...
}
```

Then you can access `pc.interfaceStore` and `pc.ovsBridgeClient` inside the function directly.